Docs: Improve Conda docs - PyPI + lock files (#5531)
nextflow-io/nextflow: docs/wave.md

```
:::

> Packages from the [Python Package Index](https://pypi.org/) can also be used by using a Conda `environment.yml` file - see the documentation about {ref}`Conda and PyPI <conda-pypi>` for details.

Reviewer: ```suggestion
Packages from the [Python Package Index](https://pypi.org/) can also be added to a Conda `environment.yml` file. See {ref}`Conda and PyPI <conda-pypi>` for more information.
```
